### PR TITLE
docs: Add documentation on how to use project

### PR DESCRIPTION
**Pull request by bgtill**
Add documentation on how to use gorm-snowflake with naming strategy. Probably pretty trivial but having a good example wouldn't hurt.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,69 @@ Notable Snowflake (SF) features that affect decisions in this driver
...
+```go
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/snowflakedb/gosnowflake"
+	snowflake "github.com/tillfinancial/gorm-snowflake"
```
Comment (omixen) [inline README.md]: change to point to this repo

File: README.md [hunk 2]
```diff
@@ -11,3 +11,69 @@ Notable Snowflake (SF) features that affect decisions in this driver
...
+		panic(err)
+	}
+
+	as := []User{}
+	tx := db.Where("status = ?", "active").Find(&as)
+	if tx.Error != nil {
+		fmt.Println(tx.Error)
+	}
+
+	bytes := errorss.RtnPanic(json.MarshalIndent(as, "", "\t"))
```
Comment (omixen) [inline README.md]: can you remove the use of error lib?

**Review by omixen — COMMENTED** [review]
hey thanks for the PR.

File: README.md (hunk 2 above)
Comment (bgtill) [reply README.md]: oops, yep.

**Review by omixen — APPROVED** [review]

**Pull request merged**